local-volume/provisioner/pkg/common/common_test.go

```diff
@@ -69,3 +69,24 @@ func TestSetupClientByInCluster(t *testing.T) {
 		t.Errorf("InClusterConfig not invoked")
 	}
 }
+
+func TestLoadProvisionerConfigs(t *testing.T) {
+	tmpConfigPath := "/tmp/provisioner/config"
+	if err := os.MkdirAll(tmpConfigPath, 0777); err != nil {
```

Comment:
There's a TempDir function you can use that will randomly generate a unique directory name, so you won't conflict with other tests.

Reply:
updated, PTAL, thanks